pkg/deployment/resources/pod_creator.go

```diff
@@ -286,6 +286,17 @@ func createArangoSyncArgs(apiObject meta.Object, spec api.DeploymentSpec, group
 	return args
 }
 
+func createArangoGatewayArgs(groupSpec api.ServerGroupSpec) []string {
+	args := []string{
+		fmt.Sprintf("-c %s", GatewayConfigFilePath),
```

Comment:
Single param?

Reply:
?

Reply:
Please, split it to:
- full param name (instead of -c)
- 2 params instead of 1 (so list with 2 elements)

Reply:
[]string{"--config...", GatewayConfigFilePath}